src/main/java/com/hazelcast/cloud/Client.java

```diff
@@ -32,11 +32,13 @@ public static void main(String[] args) {
 
         IMap<String, String> map = client.getMap("map");
         Random random = new Random();
+        byte iterationCounter = 0;
```

Comment:
I think it would be better to use `int` here instead of `byte` to avoid implicit casting in the ++ operator from `int` to `byte`

Reply:
Makes sense, thanks.